lib/win32.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+const pify = require('pify');
+const fs = require('fs');
+const path = require('path');
+const readdir = pify(fs.readdir);
+const stat = pify(fs.stat);
+const setAttributes = pify(require('fswin').setAttributes);
+
+const showModules = (projectPath, deps, showAll) =>
+  readdir(path.join(projectPath, 'node_modules'))
+    .then((files) => Promise.all(files.map(
+      (dep) => {
+        const depPath = path.join(projectPath, 'node_modules', dep);
+
+        return stat(depPath).then(
+          (stats) => stats.isDirectory() && {path: depPath, hidden: deps.indexOf(dep) === -1 && !showAll}
+        );
+      })
+    ).then(
+        (files) => Promise.all(
+          files.map((item) => setAttributes(item.path, {IS_HIDDEN: item.hidden}))
+        )
+      )
+    );
+
+const handleDeps = (rootPackage, opts) => {
+  let actions = [];
+  if (opts.clear) {
+    actions.push(showModules(opts.projectPath, [], true));
+  } else {
+    // TODO: Support optional dependencies
```

Comment:
Optional dependencies should already be in your dependencies object, because of normalization from [`read-pkg-up`](https://github.com/sindresorhus/read-pkg-up).